web/templates/user_profile_modal.hbs

```diff
@@ -61,11 +61,7 @@
                                     {{/if}}
                                 </div>
                             </div>
-                            <div class="col-wrap col-right">
-                                <div id="avatar" {{#if user_is_guest}} class="guest-avatar" {{/if}}
-                                  style="background-image: url('{{user_avatar}}');">
-                                </div>
-                            </div>
+                            <div class="col-wrap col-right"> <div id="avatar" {{#if user_is_guest}} class="guest-avatar" {{/if}} style="background-image: url('{{user_avatar}}');" {{#if is_bot}} class="bot-avatar" {{/if}}></div></div>
```

Comment:
This would be easier to read if the divs were in different lines and the class `if` conditions were next to each other.

Reply:
Thanks for the feedback. 
This has been fixed.